coderadar-test/src/main/asciidoc/restapi.adoc

```diff
@@ -603,9 +533,9 @@ You can list all metrics that have been measured for your project by calling the
 
 [NOTE]
 =====
-Metrics will become available as soon as you have provided an <<creating-analyzing-job,Analyzing Job>> for your project.
-At that moment, analysis of each commit defined in the job starts. Depending on how many commits that are,
-analysis will take a while.
+Metric values will become available as soon as an <<start-analyzing-job,Analyzing Job>> starts for your project.
+At that moment, analysis of each commit defined in the job starts. Depending on how many commits that are and how
```

Comment:
Commits aren't actually defined in the "job" though. Maybe a few words about how only commits in the date-range set for the project will be analyzed.